PWGMM/Tasks/CMakeLists.txt

```diff
@@ -1,30 +1,35 @@
-# Copyright 2019-2020 CERN and copyright holders of ALICE O2.
-# See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
-# All rights not expressly granted are reserved.
+#Copyright 2019 - 2020 CERN and copyright holders of ALICE O2.
```

Comment:
What are all these extra white spaces? CMakeLists.txt is not checked by clang-format, so you'll need to fix this yourself. Probably your text editor uses tabs, which is not a good idea for our codebase, that only uses normal spaces.

Reply:
I don't know, I have these when I do "clang-format -style=file -i <file>", which I am obligated to do to have the right Clang format to make this pull request :( 
I don't have them in my editor (atom), they only appear when I run this command

Reply:
I see, please fix the whitespace and do not apply clang-format to CMakeListst.txt, otherwise this seems complete.

Reply:
There is still some extra whitespace in the license header - it is required to be exact. The rest is less important, but please make sure that for `CMakeLists.txt` only your additional workflow entries appear in diff.

Reply:
Ok sorry I didn't notice